orchagent/fdborch.cpp

```diff
@@ -92,8 +99,16 @@ bool FdbOrch::storeFdbEntryState(const FdbUpdate& update)
              */
             if (port.m_bridge_port_id == it->second.bridge_port_id)
             {
-                SWSS_LOG_INFO("FdbOrch notification: mac %s is duplicate", entry.mac.to_string().c_str());
-                return false;
+                if (it->second.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)
+                {
+                    mac_move=true;
```

Comment:
The `mac_move` is set `true` in line 113 below. I believe one of these is excessive.

Reply:
Same for `oldFdbData`. Please double check.